src/main/webapp/style.css

```diff
@@ -21,12 +27,13 @@ h1 {
 
 h2,
 h3 {
-  color: #856c8b;  
+  background-color: #ececec;
```

Comment:
The background color of the body tag is already this color. Is there a reason that you specify it again here?